### PR TITLE
port over form related scss, delete some old bootstrap3 artifacts

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Ports over the still relevant form scss from webpack; also removes a couple things from the bootstrap 3 era that aren't relevant anymore. (see inline notes)

This pull request makes the following changes:
* ports form-related scss to its own module, or the appropriate patient module
* deletes some old webpack 3 styling

no view changes

It relates to the following issue #s: 
* Bumps #1854 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -89,65 +79,22 @@ a:hover, a:active, a:visited, a:focus {
...
-  @extend .text-center;
-  }
-.form {
-  @extend .col-md-6;
-  }
-.form-centered {
-  @extend .col-md-6;
-  @extend .text-center;
-  }
-.submit {
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss:120]: just moved, fake diff

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -89,65 +79,22 @@ a:hover, a:active, a:visited, a:focus {
...
-  font-size: $font-size;
-}
 
 .btn-lg {
   margin: 15px 0;
   padding: 15px 30px;
 }
 
 
-.column {
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss:109]: bootstrap 3

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -89,65 +79,22 @@ a:hover, a:active, a:visited, a:focus {
...
-  @extend .col-md-6;
-  @extend .text-center;
-  }
-.form {
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss:113]: bootstrap 3

File: app/assets/stylesheets/application.css.scss [hunk 4]
```diff
@@ -89,65 +79,22 @@ a:hover, a:active, a:visited, a:focus {
...
-  @extend .col-md-6;
-  }
-.form-centered {
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss:116]: bootstrap 3

File: app/assets/stylesheets/application.css.scss [hunk 5]
```diff
@@ -89,65 +79,22 @@ a:hover, a:active, a:visited, a:focus {
   text-decoration: none;
 }
 
-.border-bottom {
-  border-bottom: 1px solid black;
-  padding-bottom: 10px;
-}
 
 
-// increase size of text in form fields
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss:98]: bootstrap 3

File: app/assets/stylesheets/application.css.scss [hunk 6]
```diff
@@ -89,65 +79,22 @@ a:hover, a:active, a:visited, a:focus {
...
-
-
 .button-xs {
 // maybe deleteable?
   @extend .btn;
   @extend .btn-primary;
   @extend .btn-sm;
 }
 
-.info-form .checkbox label {
```
Comment (colinxfleming) [inline app/assets/stylesheets/application.css.scss:138]: Falls under 'didn't do anything'

**Review by colinxfleming — COMMENTED** [review]
notes

**Review by lomky — APPROVED** [review]
🎇 👨🏼‍🎨

**Pull request merged**